lib/index.js

```diff
@@ -309,6 +309,7 @@ module.exports = class Coins {
     convertFunds (data, cb) {
         return this._request({
             url: "crypto-exchanges"
+          , method: "POST"
```

Comment:
Thanks! Indeed, it's a [`POST`](https://coins.readme.io/docs/crypto-exchange).

Can you fix the url in the comment too, please?